models/user/search.go

```diff
@@ -62,6 +64,13 @@ func (opts *SearchUserOptions) toSearchQueryBase() *xorm.Session {
 		cond = cond.And(builder.Eq{"id": opts.UID})
 	}
 
+	if opts.SourceID > 0 {
```

Comment:
this should only be exposed to admins, as login source seems to be rather sensitive information.

Reply:
Done.